src/test/groovy/life/qbic/utils/BioinformaticAnalysisSpec.groovy

```diff
@@ -0,0 +1,62 @@
+package life.qbic.utils
+
+import spock.lang.Specification
+import java.nio.file.NotDirectoryException
+import java.nio.file.Paths
+import java.text.ParseException
+
+/**
+ *  Tests for the BioinformaticAnalysisParser
+ *
+ * @since 1.8.0
+ * @see BioinformaticAnalysisParser
+ *
+ */
+class BioInformaticAnalysisSpec extends Specification {
+
+    def exampleDirectoriesRoot = this.getClass().getResource("/dummyFileSystem/bioinformatic-analysis-output").getPath()
+    BioinformaticAnalysisParser bioinformaticAnalysisParser = new BioinformaticAnalysisParser()
+
+    def "parsing a valid file structure returns a Json String"() {
+        given:
+        def pathToDirectory = Paths.get(exampleDirectoriesRoot, "validates")
+        when:
+        bioinformaticAnalysisParser.parseFrom(pathToDirectory)
+        then:
+        noExceptionThrown()
+    }
```

Comment:
Just because no exception has been thrown, that does not mean that it worked.

Check for:

1. The correct object type
2. Valid fields reflecting data structure that has been passed to the parser

Reply:
Good Catch, thanks to this i found an issue with the nfcoreParser 👀